src/domain/thirdpartyRequests/transactions.ts

```diff
@@ -176,7 +176,63 @@ async function forwardTransactionRequestError(
   }
 }
 
+/**
+ * @function forwardTransactionRequestNotification
+ * @description Forwards a PATCH transactions requests to destination PISP for processing
+ * @param {string} path Callback endpoint path
+ * @param {HapiUtil.Dictionary<string>} headers Headers object of the request
+ * @param {RestMethodsEnum} method The http method PATCH
+ * @param {string} payload Body of the received kafka commit message.
+ * @throws {FSPIOPError} Will throw an error if no endpoint to forward the transactions requests is
+ * found, if there are network errors or if there is a bad response
+ * @returns {Promise<void>}
+ */
+async function forwardTransactionRequestNotification(
+  headers: Hapi.Util.Dictionary<string>,
+  transactionRequestId: string,
+  payload: string,
+  path: string,
+  method: RestMethodsEnum,
+  ): Promise<void> {
+
+  // todo: this is a temporary interim endpoint we are using until a PATCH TPR transaction endpoint is added.
+  //       i.e TP_CB_URL_TRANSACTION_REQUEST_PATCH
+  const endpointType = Enum.EndPoints.FspEndpointTypes.TP_CB_URL_TRANSACTION_REQUEST_POST
+  const fspiopSource: string = headers[Enum.Http.Headers.FSPIOP.SOURCE]
+  const fspiopDestination: string = headers[Enum.Http.Headers.FSPIOP.DESTINATION]
+
+  try {
+    const endpoint = await Util.Endpoints.getEndpoint(
+      Config.ENDPOINT_SERVICE_URL,
+      fspiopDestination,
+      endpointType)
+    Logger.info(`transactions::forwardTransactionRequestNotification -
+      Resolved PISP party ${endpointType} endpoint for transactionRequest
+      ${transactionRequestId} to: ${inspect(endpoint)}`)
+
+    const fullUrl: string = Mustache.render(endpoint + path, { ID: transactionRequestId })
+    Logger.info(`transactions::forwardTransactionRequestNotification -
+      Forwarding transaction request to endpoint: ${fullUrl}`)
+
+    await Util.Request.sendRequest(
+      fullUrl,
+      headers,
+      fspiopSource,
+      fspiopDestination,
+      method,
+      payload,
+      Enum.Http.ResponseTypes.JSON,
+      // todo: is span something we need to handle?
+      null)
+
+  } catch (err) {
+    // todo: do we need forward any errors here and to who?
```

Comment:
Yeah this is the trick with async apis - I think we should send a `PUT /thirdpartyRequests/transactions/{id}/error` to the PISP - but since we're only worried about happy path this is fine for now. 

Maybe for now can we add a log here so it's easier to track this down?

Reply:
added todo. added log.